ecmd-core/ext/template/capi/makefile.capi

```diff
@@ -60,6 +61,9 @@ install:
         endif
 	@echo "Installing ${EXTENSION_NAME_u} eCMD Client Extension includes to ${INSTALL_PATH}/ext/${EXTENSION_NAME}/capi/ ..."
 	cp ${CAPI_INCLUDES} ${INSTALL_PATH}/ext/${EXTENSION_NAME}/capi/.
+	@echo "Installing Autogenerated header files to ${INSTALL_PATH}/capi/ ..."
+	@cp ${EXTRA_INCLUDES} ${INSTALL_PATH}/capi/.
```

Comment:
Copying the required autogenerated header file to INSTALL_PATH/capi instead of seperate directory